appmap-java-maven-plugin/pom.xml

```diff
@@ -0,0 +1,88 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.appland</groupId>
+    <artifactId>appmap-java-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <version>0.5.0-SNAPSHOT</version>
+    <name>Appland Java Recorder Maven Plugin</name>
+    <description>This maven plugin helps you automatically generate an Appland AppMap using the Java Recorder.</description>
+    <url>https://github.com/applandinc/appmap-java/tree/master/java-maven-plugin</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.appland.appmap</groupId>
+            <artifactId>java-agent</artifactId>
+            <version>0.5.0</version>
+            <scope>system</scope>
+            <systemPath>C:/Users/msanchez3/.m2/repository/com/appland/appmap/java-agent/0.5.0/java-agent-0.5.0.jar</systemPath>
```

Comment:
I believe this is the last outstanding item to be removed and then we can merge. Thank you @marcuss!

Reply:
I left this here until the agent gets deployed in maven central? is this already done? @dustinbyrne

Reply:
It's not, but we should be getting to this next week. It should also be possible to install the agent as a maven package and pull it from the local repository.

https://maven.apache.org/guides/mini/guide-3rd-party-jars-local.html

Reply:
OK removing this, setting the PR as ready to merge.